owl/SOMA-HOME.owl

```diff
@@ -184,6 +184,15 @@
     
 
 
+    <!-- http://www.ease-crc.org/ont/SOMA-HOME.owl#Kitchen -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/SOMA-HOME.owl#Kitchen">
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/SOMA.owl#Room"/>
+        <rdfs:comment rdf:datatype="http://www.w3.org/2001/XMLSchema#string">A designated room to perform the task of cooking</rdfs:comment>
```

Comment:
What type of dispositions does it have? I think we should explicate that ;)
Would be actually nice for all concepts here.

Reply:
any progress?

Reply:
@kaviyachandran ?

Reply:
Hi Daniel, I have updated the PR with dispositions for the objects

Reply:
@kaviyachandran the pull request still has the status "requested changes must be addressed". you need to "re-request review" to trigger another review.
But it seems conflicts popped up that need to be solved first.

Reply:
Hi Daniel, I fixed the conflicts.

Reply:
The containment disposition should go to the room concept itself. What would be a disposition of a kitchen, specifically? This question @daniel86 too.

E.g., both kitchens and bathrooms tend to be what in Germany is called nassbereich. I.e., these rooms should be able to handle a larger level of moisture than other rooms of a house (and there are restrictions on what construction materials to use for the wall and floor coverings as a result).

There is of course also the fact that the kitchen is the room intended for cooking :) I'm not sure about whether this is a disposition of the room itself or a result of various devices and tools being typically located there.